static/js/reactions.js

```diff
@@ -270,7 +270,7 @@ exports.view.insert_new_reaction = function (opts) {
 
     if (opts.reaction_type !== 'unicode_emoji') {
         context.is_realm_emoji = true;
-        context.url = emoji.all_realm_emojis[emoji_code].emoji_url;
+        context.url = emoji.all_realm_emojis[emoji_name].emoji_url;
```

Comment:
I'm not sure what the bug is here, but with this commit, when I add a realm emoji (e.g. the default `green_check`) reaction, the count immediately appears as 2.  Since I'd like to be able to merge incrementally, we should figure out what that is.

Reply:
Fixed it! Actually, moved the fix from second last commit to this commit.